test-app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
-    "build": "NODE_ENV=production webpack --config webpack.config.js",
-    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.js"
+    "build": "NODE_ENV=production webpack --config webpack.config.babel.js",
+    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.babel.js"
   },
   "devDependencies": {
```

Comment:
Please add `{ "engines": { "node": ">= 5.0" } }` and this is ready for merging :)